css/styles.css

```
@@ -618,44 +983,127 @@ body {
   padding-left: 72px;
   padding-top: 45px;
   padding-bottom: 30px;
-
 }
 
 .product-card {
   display: grid;
   grid-template-columns: 217px 160px 160px 520px;
-  grid-template-rows: 29px 55px 37px;
+  grid-template-rows: min-content 1fr min-content;
   grid-template-areas:
-  "product-image product-title product-title rating-stars"
-  "product-image miniblock-left miniblock-right rating-stars"
-  "product-image button-showmore button rating-text";
+    "product-image product-title product-title rating-stars"
+    "product-image miniblock-left miniblock-right rating-stars"
+    "product-image button-showmore button rating-text";
   padding-top: 30px;
   padding-bottom: 30px;
+  row-gap: 4px;
+  border-top: 1px solid #e6e6e6;
+}
+
+.product-card:last-child {
+  border-bottom: 1px solid #e6e6e6;
 }
 
 .product-card-image-link {
   grid-area: product-image;
   margin-right: 37px;
 }
 
+.product-card-image:hover {
+  opacity: 0.6;
+}
+
+.product-card-image:focus {
+  outline: 3px solid #83B3D3;
+}
+
+.product-card-image:active {
+  opacity: 0.3;
+}
+
 .product-card-title-link {
   grid-area: product-title;
 }
 
+.product-card-title:hover {
+  color: #756257;
+}
+
+.product-card-title:focus {
+  color: #756257;
+  outline: 3px solid #83B3D3;
+  border-radius: 10px;
+}
+
+.product-card-title:active {
+  color: #756257;
+  opacity: 0.3;
+}
+
 .product-card-title {
   font-size: 24px;
   line-height: 31px;
   margin: 0;
+  margin-left: 15px;
+  padding-right: 15px;
+  width: max-content;
 }
 
 .product-card-button {
-  grid-area:  button;
+  grid-area: button;
+}
+
+.button-showmore {
+  grid-area: button-showmore;
+}
+
+.button-showmore:hover {
+  background-color: #615048;
+}
+
+.button-showmore:focus {
+  background-color: #615048;
+  outline: 3px solid #83B3D3;
+}
+
+.button-showmore:active {
+  background-color: #615048;
+  opacity: 0.3;
 }
 
 .rating-stars {
   grid-area: rating-stars;
-  justify-items: end;
-  margin: 0;
+}
+
+.rating-four-star {
+  background-image: url("../img/star.svg");
+  background-repeat: repeat;
+  background-size: 18px 17px;
+  width: 73px;
+  height: 18px;
+  margin-left: auto;
+  margin-right: 40px;
+  margin-top: 4px;
+  padding: 0;
+}
+
+.rating-three-star {
+  background-image: url("../img/star.svg");
+  background-repeat: repeat;
+  background-size: 18px 17px;
```

Ты можешь сделать так:

```suggestion
  background-size: 25px 17px;
```

Это сработает, если у тебя есть viewbox в иконке. Он не позволит `svg` исказиться, а лишнее расстояние как раз создаст пробелы между звездами.

Кроме этого есть еще и значение `space` у `background-repeat`. Можно с ним поработать, но скорее всего точности добиться не выйдет

---

огонь!круто получилось!